package.json

```diff
@@ -43,7 +43,7 @@
     "test-node-cli-mode": "bash < scripts/test-node-cli-mode.sh",
     "test-std-in": "bash < scripts/test-std-in.sh",
     "test-react": "jest scripts/test-react",
-    "test": "yarn test-residual && yarn test-serializer && yarn test-sourcemaps && yarn test-error-handler && yarn test-std-in && yarn test-test262 && yarn test-react && yarn test-internal",
+    "test": "yarn test-residual && yarn test-serializer && yarn test-sourcemaps && yarn test-error-handler && yarn test-std-in && yarn test-test262 && yarn test-internal && yarn test-react",
```

Comment:
Any reason for this change?

Reply:
test-react writes to console.error, which cases yarn test to exit early. Unrelated to the rest of this pull request, but something that needs a work around until test-react gets fixed.